### DISCUSSION TITLE
Backslash is not recognized

### DISCUSSION THREAD

**Discussion by ghost** [cypress-io/cypress#14650] [Questions and Help] [answered]
Hello all, 

I am trying to connect to a SQL Server with Cypress and have configuration added in the cypress.json as described in the documentations, unfortunately I am not able to connect to our database, I thought it is due to VPN issues as our database(has 2 VPN's) is not accessible from our Local systems. But now I realized that the backward Slash '\' is not recognized in the cypress Dashboard Settings ->Configuration and skips the first letter of my username i.e.,
My username given for Database in Cypress.json : **CYPRESS\nshashi**, in the Cypress Dashboard Settings ->Configuration it is taken as **"CYPRESS shashi".** 
I could guess the problem is because it interferes with system values but I really don't know what to do.
Could some one please help me in overcoming this issue, I am no expert in the cypress or configurations related things for Automation.

![image](https://user-images.githubusercontent.com/48453738/105176895-2c985e00-5b26-11eb-9706-a159dc9ee4b5.png)
![image](https://user-images.githubusercontent.com/48453738/105177119-8436c980-5b26-11eb-927e-b635371dec22.png)

**Comment by bahmutov** [comment, marked as answer]
When writing JSON strings you need to escape the `\` character, so it should be written as `"userName": "me\\here"`

**Reply by ghost** [reply]
Thank you, it is now interpreted correctly 👍 :-)
